DER/DerivativesContracts/Forwards.rdf

```diff
@@ -85,94 +98,68 @@
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">forward cash settlement terms set</rdfs:label>
-		<skos:definition xml:lang="en">&apos;This is the second of the two currency flows that define a single leg of a standard foreign exchange transaction.&apos;</skos:definition>
-	</owl:Class>
-	
 	<owl:Class rdf:about="&fibo-der-der-fwd;ForwardContract">
 		<rdfs:subClassOf rdf:resource="&fibo-der-drc-bsc;OTCInstrument"/>
-		<rdfs:subClassOf rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FinancialInstruments/FinancialInstruments/DerivativeInstrument"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;DerivativeInstrument"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-fwd;establishes"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementCommitment"/>
+				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasBuyer"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardContractBuyer"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-fwd;establishes.1"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardDeliveryCommitment"/>
+				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasSeller"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardContractSeller"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasBuyer"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardContractBuyer"/>
+				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasValuationTerms"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardValuationTermsSet"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasSeller"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardContractSeller"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;hasContractualElement"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementCommitment"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasValuationTerms"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardValuationTermsSet"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;hasContractualElement"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardDeliveryCommitment"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-fwd;side"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementTermsSet"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;confers"/>
```

Comment:
I don't see we need both this and hasContractualElement (above). At least have a new property that is a subPropertyOf both.